howtos/how-to-add-a-create_new_chat-button-in-talkjs/create-new-chat.js

```diff
@@ -0,0 +1,129 @@
+const $targetEl = document.getElementById('crud-modal');
+const modal = new Modal($targetEl);
+const appId = "<Add your App ID here!>";
+
+Talk.ready.then(function () {
+    const me = new Talk.User({
+        id: '123457',
+        name: 'Alice',
+        email: 'alice@example.com',
+        photoUrl: 'https://talkjs.com/images/avatar-1.jpg',
+        role: 'default',
+        welcomeMessage: 'Hey there! How are you? :-)',
+    });
+    window.talkSession = new Talk.Session({
+        appId: appId,
+        me: me,
+    });
+    const other = new Talk.User({
```

Comment:
I'd just remove the "other" user now and add the example users to the conversation. It's a bit weird having an extra user that doesn't appear in the dropdown